src/lib/form-field/_form-field-theme.scss

```diff
@@ -56,7 +56,7 @@
   .mat-form-field-underline {
     background-color: $underline-color;
 
-    &.mat-disabled {
+    .mat-form-field-disabled & {
```

Comment:
I think we have a rule that `&` can only go at the beginning in theme files, so that people can do stuff like this

```scss
.dark-theme {
  @include angular-material-theme($dark-theme);
}
```